buku

```diff
@@ -5276,10 +5278,12 @@ def main():
     """Main."""
     global ID_STR, ID_DB_STR, MUTE_STR, URL_STR, DESC_STR, DESC_WRAP, TAG_STR, TAG_WRAP, PROMPTMSG
     # readline should not be loaded when buku is used as a library
+    import readline
     try:
-        import readline
+        import colorama
+        colorama.just_fix_windows_console()  # noop on non-Windows systems
```

Comment:
ANSI colour codes are disabled by default on Windows 7-10, but are fairly easy to turn on.

[This does not affect non-Windows systems, by design.](https://github.com/tartley/colorama/blob/0.4.6/colorama/initialise.py#L75-L76)